.ci/ci.yml

```diff
@@ -123,16 +122,24 @@ jobs:
             resource: magic-modules-new-prs
             passed: [mm-generate]
             version: every
+          # TODO(ndmckinley): This will unconditionally attempt to create a PR - before
```

Comment:
Does this mean that if a change is made that only affects chef, that an empty PR would be opened against TF?

Reply:
👍